src/pages/boardView.js

```diff
@@ -140,7 +140,7 @@ function BoardView() {
                   subTasks={boardData.subTasks}
                   handleSortList={onSortListClick}
                 />
-                {!index ? <NewTask /> : null}
+                {!index ? <NewTask onNewTask={() => {}} /> : null}
```

Comment:
what's the point of () => {}?

Reply:
Yeah I can add some commenting for this. It is because the prop "onNewTask" is only applicable when NewTask is a child of the TaskList component (as it is on the HomePage and DashboardView) - on the boardView this prop can be an empty function because there is no need to re-render the tasks (there are no tasks on the boardView)

Reply:
Cool beans - thanks for addressing this comment! LGTM